.github/workflows/copy-updated-schema.yml

```diff
@@ -0,0 +1,49 @@
+name: Sync Schema Update to Website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+     # Monitor changes in this v3 file
+      - schemas/3.0.0-without-$id.json
+      - schemas/3.0.0.json
+
+jobs:
+  sync-schema:
+    runs-on: ubuntu-latest
+    environment: GH_TOKEN
+    steps:
+      - name: Checkout spec-json-schema repo
+        uses: actions/checkout@v3
+
+      - name: Checkout website repo
+        uses: actions/checkout@v3
+        with:
+          repository: asyncapi/website
+          token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          path: ./website
+
+      - name: Copy version 3 Schema File
+        run: |
+          cp schemas/3.0.0-without-\$id.json website/config
```

Comment:
will it work if file already exists in the folder?

also now in config is `3.0.0.json` - different name - I guess you want to keep it that way

Reply:
`3.0.0.json` is better name I guess, we do not need to keep original `3.0.0-without-$id.json`